org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "/app/fireflyluciferin/bin/FireflyLuciferin",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=all",
+    "--filesystem=home",
+    "--env=LD_LIBRARY_PATH=/app/lib",
+    "--env=GTK_THEME=Yaru:dark",
```

Comment:
```suggestion
```

nor this

Reply:
what do you mean for nor this?

Reply:
that thing help me choose the right GTK theme, why should I remove it?

Reply:
"the right GTK theme" that's acutally incorrect. I don't have that theme and I use the stock distro.

Reply:
that is a preference, if you have that theme, you'll use it, if you don't have it it will fallback to the default one.
isn't that ok?

Reply:
no.

Reply:
is there a better way to tell my app to use a dark theme for the GTK libs?

Reply:
fixed